projects/storefrontapp/src/app/app.module.ts

```diff
@@ -38,6 +39,8 @@ if (!environment.production) {
 
     ...cdsFeature.imports,
 
+    ...productconfigFeature.imports,
```

Comment:
This is a bit outdated mechanism, please check the new one that we are using: https://github.com/SAP/spartacus/blob/develop/projects/storefrontapp/src/app/app.module.ts#L33